timemachine/potentials/bonded.py

```diff
@@ -1,18 +1,26 @@
 import jax.numpy as np
 
-def centroid_restraint(conf, params, box, lamb, masses, group_a_idxs, group_b_idxs, kb, b0):
+def centroid_restraint(conf, params, box, lamb, group_a_idxs, group_b_idxs, kb, b0):
 
```

Comment:
```suggestion
    """Computes kb  * (r - b0)**2 where r is the distance between the centroids of group_a and group_b
    
    Notes
    ------
    * Geometric centroid, not mass-weighted centroid
    * Gradient undefined when `r - b0 == 0` and `b0 != 0` (explicitly stabilized in case `b0 == 0`)
    """
```

Add docstring